pkg/server/sandbox_run.go

```diff
@@ -78,9 +78,45 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 	if err != nil {
 		return nil, fmt.Errorf("failed to get sandbox image %q: %v", c.sandboxImage, err)
 	}
+	//Create Network Namespace if it is not in host network
+	hostNet := config.GetLinux().GetSecurityContext().GetNamespaceOptions().GetHostNetwork()
+	if !hostNet {
+		// If it is not in host network namespace then create a namespace and set the sandbox
+		// handle
+		err := sandbox.CreateNetNS()
+		if err != nil {
+			return nil, err
+		}
+		defer func() {
+			if retErr != nil {
+				if err := sandbox.RemoveNetNS(); err != nil {
+					glog.Errorf("failed to remove network namespace for sandbox %q: %v", id, err)
```

Comment:
s/failed/Failed.

Not your fault, but let's fix it here.

Our convention in Kubernetes is that, error must start with lower case; log must start with upper case.